libs/langchain/langchain/output_parsers/pydantic.py

```diff
@@ -51,3 +55,8 @@ def get_format_instructions(self) -> str:
     @property
     def _type(self) -> str:
         return "pydantic"
+
+    @property
+    def OutputType(self) -> Type[T]:
```

Comment:
Actually should this be overriding get_output_schema instead?

Reply:
Not sure! That's what the parent parsers do

and in Runnables:


https://github.com/langchain-ai/langchain/blob/aad4865e99ea35be12043b3b46c026bb968a72f8/libs/core/langchain_core/runnables/base.py#L257-L257

https://github.com/langchain-ai/langchain/blob/aad4865e99ea35be12043b3b46c026bb968a72f8/libs/core/langchain_core/runnables/base.py#L324-L324

It seems like I'm meant to over-ride `OutputType` -- or do you think that's wrong?

Reply:
actually this is fine, leave as is